configs/mirrors.txt

```diff
@@ -1,14 +1,11 @@
 url='http://resources.ovirt.org/pub',whitelist=True
-url='http://mirror.linux.duke.edu/ovirt/pub'
 url='http://mirrors.ibiblio.org/ovirt'
 url='http://mirror.isoc.org.il/pub/ovirt'
-url='http://www.gtlib.gatech.edu/pub/oVirt/pub'
 url='http://ftp.plusline.net/ovirt'
 url='http://mirrors.rit.edu/ovirt'
 url='http://mirror.slu.cz/ovirt'
 url='http://ftp.snt.utwente.nl/pub/software/ovirt'
 url='http://ftp.nluug.nl/os/Linux/virtual/ovirt'
 url='http://ovirt.repo.nfrance.com'
 url='http://mirror.rackspace.com/ovirt'
-url='http://mirror.massclouds.com/ovirt'
```

Comment:
this one doesn't work for me as well

Reply:
I wrote to admin yesterday, so I wanted to wait for their answer for a day or two. That's why MR is draft state

Reply:
ok makes sense

Reply:
Massclouds admin replied finally - they had a problem due to migration of the server, sync is restored. I restored this mirror in configs